app.js

```diff
@@ -1,7 +1,12 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 8080;
```

Comment:
Nice. Moving these lines up here means our express app knows what it needs before it starts to run